01-http-requests/src/main/java/com/ewencluley/javatasks/httprequests/HandlerPostRqst.java

```diff
@@ -0,0 +1,49 @@
+package com.ewencluley.javatasks.httprequests;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+
+public class HandlerPostRqst implements HttpHandler {
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+
+        // Getting all the exchanged bytes and passing it to a string on the following line
+        byte[] exchangedData = exchange.getRequestBody().readAllBytes();
+        String[] exchangedDataToStringLines = new String(exchangedData, StandardCharsets.UTF_8).split("\n");
+
+        StringBuilder exchangedDataToString = new StringBuilder();
+
+        // get the numbers from each line,
+        // multiply the resulting number by 2
+        for (String s : exchangedDataToStringLines) {
+            String number = s.replaceAll("\\s*\\D+\\s*","");
```

Comment:
this removes all of the spaces from the string to leave just the numbers, however have you considered the case where the input contains multiple separate numbers, e.g. `A dozen is 12, a baker's dozen is 13`.

https://stackoverflow.com/a/19737857/10386635 might be a better way of doing this.